src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+       action="/search"
+       method="get"
+       data-qa="big"
+       class="form"
+       >
+      <label
+         for="big icon">
+        <img
+           src="./images/Search.svg"
+           class="icon--cursor icon-big"
+           alt="big Icon"
+   >
+      </label>
+
+      <input
+         type="text"
+         class="input-big input-big--appearance"
+         data-qa="keypress"
+         placeholder="Try &quot;Los Angeles&quot;"
+         id="big icon"
+           >
+    </form>
+
+    <form
+       action="/search"
+       method="get"
+       data-qa="small"
+       class="form"
+  >
+      <label
+         for="small icon"
+        >
+        <img
+           src="./images/Search.svg"
+           class="icon--cursor icon-small"
+           alt="Small Icon"
+         >
+      </label>
+      <input
+         type="text"
+         class="input-small input-small--appearance"
```

Comment:
this class can be just `input input--small` with addition modifier class for small one